src/main/java/FrameworksDrivers/CompareGraphics.java

```diff
@@ -0,0 +1,136 @@
+        setVisible(true);
+
+    }
+
+    public void paint(Graphics g) {
+        super.paint(g);
+
+        g2g = g;
+
+        // creating custom colors
+        Color color0 = Color.white;
+        Color color1 = new Color(110, 205, 250);
+        Color color2 = new Color(39, 208, 245);
+        Color color3 = new Color(0, 210, 231);
+        Color color4 = new Color(0, 211, 207);
+        Color color5 = new Color(0, 210, 175);
+        Color color6 = new Color(0, 207, 137);
+        Color color7 = new Color(65, 202, 94);
+
+        Color[] colorsList = {color0, color1, color2, color3, color4, color5, color6, color7};
+
+        String[] titles = {"Time", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"};
+
+        // drawing the day of week header boxes & outlines
+        for (int x = 0; x < 8; x++) {
+            g2g.setColor(colorsList[x]); // different color for each day slot
+            g2g.fillRect(25 + (x * 100), 100, 100, 40);
+            g2g.setColor(Color.BLACK);
+            g2g.drawString(titles[x],53 + (x * 100), 125);
+        }
+//
+        g2g.setColor(Color.white);
+        for (int x = 0; x < 8; x++) {
+            g2g.drawRect(25 + (x * 100), 100, 100, 40);
+        }
+//
+        // drawing the empty schedule outlines
+        for (int x = 0; x < 8; x++) {
+            for (int i = 0; i < 24; i++) {
+                g2g.drawRect(25 + (x * 100), 140 + (i * 25), 100, 25);
+            }
+        }
+
+        for (int x = 0; x < 24; x++) {
+            g2g.setColor(Color.gray);
+            g2g.fillRect(25, 140 + (x * 25), 100, 25);
+            g2g.setColor(Color.BLACK);
+            g2g.drawString(timesList[x], 50, 160 +(25 * x));
+        }
+        for (String day : manager.getSchedule(user1).getSchedule().keySet()){
+            for (Event e : manager.getSchedule(user1).getDayEvents(day)){
+                paintEvent(e.getEventDay(), e.getEventStartTime(), e.getEventEndTime());
+            }
+        }
+        for (String day : manager.getSchedule(user2).getSchedule().keySet()){
+            for (Event e : manager.getSchedule(user2).getDayEvents(day)){
+                paintEvent(e.getEventDay(), e.getEventStartTime(), e.getEventEndTime());
+            }
+        }
+    }
+
+    public void paintEvent(String eventDay, int eventStartTime, int eventEndTime){
+
+        String[] titles = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"};
+        // is titles iterable??
+
+        Integer[] timesList = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19,
+                20, 21, 22, 23, 24};
+
+        int xCord = 0;
+        for (int i = 0; i < 7; i++){
+            if (Objects.equals(eventDay, titles[i])){
+                xCord = 125 + (i * 100);
+            }
+        }
+
+        int yStartCord = 0;
+        for (int i = 0; i < 25; i++){
+            if (Objects.equals(eventStartTime, timesList[i])){
```

Comment:
shouldn't be using objects.equals -- if these are integers, just use == and if they're strings, use String. equals or whatever the method is

Reply:
same for line 124

Reply:
I used == to begin with, but this was what intelliJ recommended.